### ISSUE TITLE
Support for client TLS certificate in tcpsock:sslhandshake

### ISSUE THREAD

**Issue by smira** [openresty/lua-nginx-module#534]
There's already an option to validate server TLS certificate using `lua_ssl_trusted_certificate`. 

It would be nice if we could provide client TLS certificate, in the same way as `ngx_http_proxy` module is doing: http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_ssl_certificate, so that there would be two new options:

 * `lua_ssl_certificate`
 * `lua_ssl_certificate_key`

These options would initialise nginx ssl context with client certificates, which would be used during ssl handshake.

**Comment by agentzh** [comment]
@smira Yes, this has been on my TODO list. But would you mind contributing a patch for it? ;)

Thanks!

**Comment by rohitjoshi** [comment]
Any patch available?

**Comment by Lekensteyn** [comment]
This appears to be fixed since March 2022 via commit 2b902657f370e392bd5066d2eafed7a3429af19e (PR #1602). There is a new `tcpsock:setclientcert` API.

**Comment by denissabramovs** [comment]
I don't see and can't find setclientcert anywhere...

**Comment by denissabramovs** [comment]
It was not merged...

**Comment by zhuizhuhaomeng** [comment]
https://github.com/openresty/lua-nginx-module#tcpsocksetclientcert
It has been merged. But do not contain in the openresty-1.21.4 Release
